spread/google.go

```diff
@@ -350,32 +366,39 @@ func (p *googleProvider) projectImages(project string) ([]googleImage, error) {
 		return cache.images, cache.err
 	}
 
-	var result struct {
-		Items []struct {
-			Description string
-			Status      string
-			Name        string
-			Family      string
+	var results []*googleImageListResult
+	var nextPageToken string
+	for {
+		var result googleImageListResult
+		v := url.Values{
+			"orderBy":   {"creationTimestamp+desc"},
+			"pageToken": {nextPageToken},
 		}
-	}
+		err := p.dofl("GET", "/compute/v1/projects/"+project+"/global/images?"+v.Encode(), nil, &result, noPathPrefix)
+		nextPageToken = result.NextPageToken
+		if err != nil {
+			return nil, &FatalError{fmt.Errorf("cannot retrieve Google images for project %q: %v", project, err)}
+		}
+		results = append(results, &result)
```

Comment:
nitpick: I think this would read cleaner (and would eliminate the double for loop below if we had `results` as a single `googleImageListResult` and here did

```suggestion
		results.Items = append(results.Items, &result.Items...)
```
I don't think we need to save the `nextPageToken` at all for outside the for loop, so this simplifies storing the results inside a single list rather than a list of structures with lists inside the structures

Reply:
also if you do take this, probably rename `results` -> `finalResults` so it reads a bit better:
```suggestion
		finalResults.Items = append(finalResults.Items, result.Items...)
```